target/scripts/helpers/sasl.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+# Support for SASL
+
+function _sasl_passwd_create
+{
+  if [[ -n ${SASL_PASSWD} ]]
+  then
+    # create SASL password
+    echo "${SASL_PASSWD}" > /etc/postfix/sasl_passwd
+    _sasl_passwd_chown_chmod
+  else
+    rm -f /etc/postfix/sasl_passwd
+  fi
+}
+
+function _sasl_passwd_chown_chmod
```

Comment:
Apply or discard:

```suggestion
function _sasl_set_passwd_permissions
```

Reply:
I'll apply this change later today, either via two commits in web UI or when I can free enough RAM on my local system (_because of laziness avoiding CLI for git_ :sweat_smile: )

Reply:
Don't forget to update also the corresponding function calls 👍